_schemaV2.graphql

```diff
@@ -11783,7 +11783,34 @@ type Partner implements Node {
   ): LocationConnection
   meta: PartnerMeta
   name: String
+  partnerArtistsMatchConnection(
+    after: String
+    before: String
+    first: Int
+    last: Int
+    page: Int = 1
+    size: Int = 10
+    term: String!
```

Comment:
should probably change this to `query` for consistency with other match endpoints, maybe `partnerArtistsSearchConnection` is a more consistent name as well?